docs/guides/StandarizedUnitsNaming.md

```diff
@@ -0,0 +1,115 @@
+# Standardowe nazewnictwo jednostek
+
+
+
+## Wstęp + rozpiska komunikacji radiowej
```

Comment:
Avoid using special characters in headings, rephrase it somehow to remove `+`. Or just leave the first part.
```suggestion
## Wstęp
```

Reply:
Corrected.